mev_inspect/schemas/classified_traces.py

```diff
@@ -48,6 +48,37 @@ class Config:
         }
 
 
+class ClassifiedTraceStrict(BaseModel):
+    transaction_hash: str
+    block_number: int
+    trace_type: TraceType
+    trace_address: List[int]
+    classification: Classification
+
+    class Config:
+        json_encoders = {
+            # a little lazy but fine for now
+            # this is used for bytes value inputs
+            bytes: lambda b: b.hex(),
+        }
+
+
+class ClassifiedCall(ClassifiedTraceStrict):
+    to_address: str
+    from_address: str
+    gas: int
+    value: int
+    gas_used: Optional[int]
+
+
+class ClassifiedCallData(ClassifiedCall):
+    protocol: Optional[Protocol]
+    function_name: str
+    function_signature: str
+    inputs: Dict[str, Any]
+    abi_name: str
+
```

Comment:
Since these are all still "Traces" I think that should still be the main noun for the object

ex:
```
StrictClassifiedTrace
ClassifiedCallTrace
ClassifiedCallDataTrace
```

Another option, reserve classified for when we actually knew what the ABI was. So it'd be
```
Trace
CallTrace
ClassifiedCallTrace
```